tests/interfaces/test_torch_qnode.py

```diff
@@ -486,8 +486,17 @@ def circuit(U, a):
     def test_differentiable_expand(self, interface, dev_name, diff_method, grad_on_execution, tol):
         """Test that operation and nested tapes expansion
         is differentiable"""
+        kwargs = dict(
+            diff_method=diff_method,
+            grad_on_execution=grad_on_execution,
+            interface=interface
+        )
         if diff_method == "spsa":
-            np.random.seed(SEED_FOR_SPSA)
+            spsa_kwargs = dict(
+                sampler_rng=np.random.default_rng(SEED_FOR_SPSA),
+                num_directions=10
+            )
+            kwargs = kwargs | spsa_kwargs
```

Comment:
```suggestion
            kwargs = {**kwargs, **spsa_kwargs}
```
Maybe? 

combining dictionaries with `|` is 3.9+, and we still have to support Python 3.8.